HPXMLtoOpenStudio/resources/constructions.rb

```diff
@@ -2032,6 +2032,16 @@ def self.create_kiva_slab_foundation(model, int_horiz_r, int_horiz_width, int_ve
       foundation.setExteriorVerticalInsulationDepth(UnitConversions.convert(ext_vert_depth, 'ft', 'm'))
     end
 
+    if ext_horiz_r && ext_horiz_width && ext_horiz_depth
```

Comment:
You shouldn't need this, as you're currently requiring all HPXML files to have these inputs. Note that we don't have similar checks anywhere else in this method.

Reply:
Addressed.